editor/src/clj/editor/gui.clj

```diff
@@ -2566,6 +2566,12 @@
                                           (make-add-handler scene parent menu-label menu-icon add-fn opts)))))))]
       (filter some? (into node-options handler-options))))
 
+(defn- unused-display-profiles [gui-scene-node-id]
+  (g/with-auto-evaluation-context evaluation-context
+    (produce-unused-display-profiles
+      {:layout-msgs (g/node-value gui-scene-node-id :layout-msgs evaluation-context)
+       :display-profiles (g/node-value gui-scene-node-id :display-profiles evaluation-context)})))
+
```

Comment:
It looks like this function could be replaced by just `(g/node-value scene :unused-display-profiles)` at the call site, or am I missing something?

Reply:
Argh yes, you are totally right!

Reply:
Fixed!